ts/models/conversation.ts

```diff
@@ -867,34 +846,51 @@ export class ConversationModel extends Backbone.Model<ConversationAttributes> {
     });
     await this.commit();
 
-    await this.queueJob(async () => {
+    void this.queueJob(async () => {
       await this.sendMessageJob(messageModel, expireTimer);
     });
   }
 
   public async bouncyUpdateLastMessage() {
-    if (!this.id) {
+    if (!this.id || !this.get('active_at')) {
       return;
     }
-    if (!this.get('active_at')) {
+    const messages = await Data.getLastMessagesByConversation(this.id, 1, true);
+
+    if (!messages || !messages.length) {
       return;
     }
-    const messages = await getLastMessagesByConversation(this.id, 1, true);
     const lastMessageModel = messages.at(0);
-    const lastMessageJSON = lastMessageModel ? lastMessageModel.toJSON() : null;
     const lastMessageStatusModel = lastMessageModel
       ? lastMessageModel.getMessagePropStatus()
       : undefined;
     const lastMessageUpdate = createLastMessageUpdate({
-      currentTimestamp: this.get('active_at'),
-      lastMessage: lastMessageJSON,
       lastMessageStatus: lastMessageStatusModel,
       lastMessageNotificationText: lastMessageModel
         ? lastMessageModel.getNotificationText()
         : undefined,
     });
-    this.set(lastMessageUpdate);
-    await this.commit();
+
+    if (
+      lastMessageUpdate.lastMessage !== this.get('lastMessage') ||
+      lastMessageUpdate.lastMessageStatus !== this.get('lastMessageStatus')
+    ) {
+      const lastMessageAttribute = this.get('lastMessage');
+      if (
+        lastMessageUpdate.lastMessageStatus === this.get('lastMessageStatus') &&
+        lastMessageUpdate.lastMessage &&
+        lastMessageUpdate.lastMessage.length > 40 &&
+        lastMessageAttribute &&
+        lastMessageAttribute.length > 40 &&
+        lastMessageUpdate.lastMessage.startsWith(lastMessageAttribute)
+      ) {
+        // if status is the same, and text have a long length which start with the db status, do not trigger an update.
```

Comment:
rebasing emoji reacts and leaving comments as I work through that, If possible I will still do a full review of this after.

comment should ready "if status is the same, and text has a long length which starts with the db status, do not trigger an update."